source/examples/xgboost-azure-mnmg-daskcloudprovider/notebook.ipynb

```diff
@@ -0,0 +1,1422 @@
+{
```

Comment:
<p>Do we use this function again? If not maybe just get rid of it and print out whatever directly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>